docs/guides/getting-started.md

```diff
@@ -168,11 +170,10 @@ Generating a transaction is done in three steps. First, call a method returning
 const intent = await org.appIntent(voting, 'vote', [votes[0].id, true, true])
 ```
 
-Then to retrieve the path you want, pass the account that will be signing the transaction. Aragon Connect will go through the permissions of the organization, and return all possible paths:
+Then to retrieve the path you want, pass the account that will be signing the transaction. Aragon Connect will go through the permissions of the organization, and return the shortest path:
 
 ```javascript
-// The first path is the shortest
-const [path] = intent.paths(wallet.account)
+const path = intent.paths(wallet.account)
```

Comment:
I can’t wait for the new API, returning a single value on a method named as a plural feels really weird. I was wondering if in the meantime, we should add an alias on `intent#path()` and only refer to `.path()` in the documentation?

Reply:
After having thought more about it: let’s not introduce any change and just move to the new API.

Reply:
As discussed offline we decide to not change the API given a major change will be introduced soon.